### ISSUE TITLE
Problem snarling chunked SV graph

### ISSUE THREAD

**Issue by glennhickey** [vgteam/vg#2066]
Context expanding over deletions in vg chunk can leave holes in the reference path.  vg chunk handles this by clipping the reference path at the first sign of discontinuity.  In certain cases, this can lead to a path whose last node is not on a graph tip in the chunk.  This in turn will cause the cactus snarl finder to not use the reference path when looking for telomeres.  Then the telomeres it does find (via pingpong) sometimes don't seem to make a terrific root for the cactus graph, as it then recurses itself into oblivion while enumerating snarls.  

tldr:  dangling ends from cutting in complex regions can lead to crashes when finding snarls in the resulting chunks.
